tests/dns/dns-query/DetectDnsQueryTest01/README.md

```diff
@@ -0,0 +1,3 @@
+# Description
+
+Translation of unit test DetectDnsQueryTest01
```

Comment:
In the DNS rust conversion this test no longer exists so we have no point a reference. A nice explanation would be good.  And yeah, I realize the unit tests didn't have much of an explanation either.

Reply:
I added the comment for each unit test + my personal analysis of what is being tested